gen_png.py

```diff
@@ -74,15 +74,28 @@ def git_commit_raster():
     print_pipe(stream.read())
 
 # create a new tag
-def create_tag():
+def create_tag(added, modified, deleted, renamed_old, renamed_new):
     """Creates a new git tag and names it appropriately"""
```

Comment:
Function documentation should get updated with sth along the lines of ", names it appropriately and creates a commit message based on the files that changed."